products/dns/api.yaml

```diff
@@ -224,6 +224,15 @@ objects:
                   name: 'ipv4Address'
                   required: true
                   description: 'IPv4 address of a target name server.'
+                - !ruby/object:Api::Type::Enum
+                  name: 'forwardingPath'
+                  description: |
+                    Forwarding path for this TargetNameServer. If unset or `default` Cloud DNS will make forwarding
+                    decision based on address ranges, i.e. RFC1918 addresses go to the VPC, Non-RFC1918 addresses go
+                    to the Internet. When set to `private`, Cloud DNS will always send queries through VPC for this target
+                  values:
```

Comment:
Just want to confirm that the API returns what we send it here- so if we send it "" it returns "" and if we send it "default" it returns "default"?

Reply:
Yep, it also unsets based on what we send in the PATCH request, so going from forwarding_path = 'private' to not setting forwarding_path in the config causes it to be removed on the API side as well